test/mocks/AmAmmMock.sol

```diff
@@ -0,0 +1,66 @@
+// SPDX-License-Identifier: AGPL-3.0
+pragma solidity ^0.8.0;
+
+import {Test, console} from "forge-std/Test.sol";
+
+import {Currency, CurrencyLibrary} from "@uniswap/v4-core/src/types/Currency.sol";
+import {PoolId} from "v4-core/types/PoolId.sol";
+import {AMAMM} from "../../src/AMAMM.sol";
+
+import "./ERC20Mock.sol";
+
+contract AmAmmMock is AMAMM {
+    using CurrencyLibrary for Currency;
+
+    ERC20Mock public immutable bidToken;
+    ERC20Mock public immutable feeToken0;
+    ERC20Mock public immutable feeToken1;
+
+    mapping(PoolId id => bool) public enabled;
+    mapping(PoolId id => uint24) public maxSwapFee;
+
+    constructor(ERC20Mock _bidToken, ERC20Mock _feeToken0, ERC20Mock _feeToken1) {
+        bidToken = _bidToken;
+        feeToken0 = _feeToken0;
+        feeToken1 = _feeToken1;
+    }
+
+    function setEnabled(PoolId id, bool value) external {
```

Comment:
Enhancement: I'm unsure why setEnabled and setMaxSwapFee are in the mock; I think they should be in the AMAMM.